svg/elements/feComposite.json

```diff
@@ -24,19 +24,19 @@
               "version_added": null
             },
             "opera": {
-              "version_added": false
+              "version_added": "15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "14"
             },
             "safari": {
-              "version_added": null
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
```

Comment:
Marked this true as there are webkit bugs against lighter for operator - which implies that feComposite must be supported.

Reply:
Will do.